src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -3,14 +3,19 @@
     const notifications = JSON.parse(notificationsContainer.dataset.notifications);
     const template = notificationsContainer.dataset.template;
     const addNotification = ({ detail }) => {
-        const { onShow, label, message } = detail;
+        const { onShow, label, message, rawPlaceholdersMap } = detail;
```

Comment:
Probably will fail if backend notification occurs. It needs the default value `{}`